src/go/k8s/kuttl-v2-test.yaml

```diff
@@ -12,10 +12,10 @@ commands:
   - command: kubectl taint nodes -l node-role.kubernetes.io/master= node-role.kubernetes.io/master:NoSchedule-
     ignoreFailure: true
   - command: ./hack/install-cert-manager.sh tests/_e2e_artifacts_v2
-  - command: bash -c "kubectl kustomize https://github.com/redpanda-data/redpanda//src/go/k8s/config/crd | kubectl apply -f -"
+  - command: bash -c "kubectl kustomize ./config/crd | kubectl apply -f -"
   - command: helm repo add redpanda https://charts.redpanda.com
   - command: helm repo update
-  - command: helm install --set scope=Namespace --set logLevel=trace --set image.tag=dev
```

Comment:
Shouldn't scope be `namespaced`?

Reply:
The default is namespace, this should not break anything.